service/controller/v23/adapter/spec.go

```diff
@@ -27,6 +27,8 @@ const (
 	// rollingUpdatePauseTime is how long to pause ASG operations after creating
 	// new instances. This allows time for new nodes to join the cluster.
 	rollingUpdatePauseTime = "PT15M"
+	// logEBSVolumeMountPoint is the path for mounting the log EBS volume
+	logEBSVolumeMountPoint = "/dev/xvdf"
```

Comment:
@giantswarm/team-spirit-pe should we give the user the possibility to specify the mount point (and so introduce the field in the spec)? Which is your feeling about this? I went with it because I think that the user does not care that much about the mount point of the log volume so we can take this decision for them.

Reply:
No user should not care at all.

Reply:
Agree with that.